lib/hooks/queries/useInfiniteMarkets.ts

```diff
@@ -78,14 +77,14 @@ export const useInfiniteMarkets = (
     const statuses = getFilterValuesByType(filters, "status") as MarketStatus[];
     const tags = getFilterValuesByType(filters, "tag");
     const currencies = getFilterValuesByType(filters, "currency");
-
+    console.log(statuses);
     const markets: Market<IndexerContext>[] = await sdk.model.markets.list({
       where: {
         categories_isNull: false,
         status_not_in: [MarketStatus.Destroyed],
         status_in: statuses.length === 0 ? undefined : statuses,
         tags_containsAny: tags.length === 0 ? undefined : tags,
-        pool_isNull: withLiquidityOnly ? false : undefined,
+        pool_isNull: true,
```

Comment:
I think this filters for only markets that has no pool? Is that what we want here?